server/monitor.go

```diff
@@ -2005,68 +2012,121 @@ func (s *Server) accountInfo(accName string) (*AccountInfo, error) {
 	}
 	a.mu.RLock()
 	defer a.mu.RUnlock()
-	claim, _ := jwt.DecodeAccountClaims(a.claimJWT) // ignore error
+	var vrIssues []ExtVrIssues
+	claim, _ := jwt.DecodeAccountClaims(a.claimJWT) //ignore error
+	if claim != nil {
+		vr := jwt.ValidationResults{}
+		claim.Validate(&vr)
+		vrIssues = make([]ExtVrIssues, len(vr.Issues))
+		for i, v := range vr.Issues {
+			vrIssues[i] = ExtVrIssues{v.Description, v.Blocking, v.TimeCheck}
+		}
+	}
 	exports := []ExtExport{}
 	for k, v := range a.exports.services {
-		e := ExtExport{
-			Export: jwt.Export{
-				Subject:      jwt.Subject(k),
-				Type:         jwt.Service,
-				TokenReq:     v.tokenReq,
-				ResponseType: jwt.ResponseType(v.respType.String()),
-			},
-			ApprovedAccounts: []string{},
-		}
-		for name := range v.approved {
-			e.ApprovedAccounts = append(e.ApprovedAccounts, name)
+		var e ExtExport
+		subj := jwt.Subject(k)
+		if v == nil {
+			e = ExtExport{
+				Export: jwt.Export{
+					Subject: subj,
+					Type:    jwt.Service,
+				},
+			}
+		} else {
+			e = ExtExport{
+				Export: jwt.Export{
+					Subject:      subj,
+					Type:         jwt.Service,
+					TokenReq:     v.tokenReq,
+					ResponseType: jwt.ResponseType(v.respType.String()),
+				},
+				ApprovedAccounts: []string{},
+			}
+			for name := range v.approved {
+				e.ApprovedAccounts = append(e.ApprovedAccounts, name)
+			}
 		}
 		exports = append(exports, e)
 	}
 	for k, v := range a.exports.streams {
+		token := false
```

Comment:
Similar to previous comment. Would create the ExtExport{} with common stuff then fill `if v != nil`

Reply:
fixed, same as above